pages/login.page.tsx

```diff
@@ -18,11 +18,7 @@ const IndexPage = (): ReactElement => (
       subtitle="MPDX is fundraising software from Cru that helps you grow and maintain your ministry
 partners in a quick and easy way."
     >
-      <Button
-        size="large"
-        variant="contained"
-        onClick={() => signin('thekey', { callbackUrl: process.env.SITE_URL })}
-      >
+      <Button size="large" variant="contained" onClick={() => signin('thekey')}>
```

Comment:
The ```callbackUrl``` isn't needed anymore because the ```serverRunTimeConfig.NEXTAUTH_URL``` now contains the URL to navigate to after auth?

Reply:
Ya it just uses the current browser url I think. Not sure why Tatai set it originally.